sql/src/test/java/org/apache/druid/sql/calcite/QueryTestRunner.java

```diff
@@ -772,8 +772,11 @@ public void run()
 
   public QueryResults resultsOnly()
   {
-    ExecuteQuery execStep = (ExecuteQuery) runSteps.get(0);
-    execStep.run();
+    for (QueryRunStep runStep : runSteps) {
+      runStep.run();
+    }
+
+    BaseExecuteQuery execStep = (BaseExecuteQuery) runSteps.get(runSteps.size() - 1);
```

Comment:
I'm not very familiar with this, why is the change required to run all runsteps for this benchmark (and not others)?

Reply:
`queryTestBuilder.results()` didn't support MSQ apparently, since it only had the execute step, and not the ExtractResultsFactory step.

So, without this change, instead of getting the query results, we were getting the query ID. The ExtractResultsFactory step (added as a custom runner when declaring QueryTestBuilder) takes the query ID, and contacts the overlord client, and fetches the actual results.

> (and not others)?

In the regular MSQ tests, we do `testBuilder().run()` which handled both steps already.